tests/test_embedding_ascend.py

```
@@ -0,0 +1,280 @@
+        import torch_npu  # noqa: F401
+    except ImportError:
+        return False
+    return hasattr(torch, "npu") and torch.npu.is_available()
+
+
+def _ascend_kernel_available() -> bool:
+    if not _npu_available():
+        return False
+    try:
+        from rl_engine.kernels.ops.ascend.linear.embedding import _NPU_EXT_AVAILABLE, _C_npu
+    except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)